TeachingRecordSystem/src/TeachingRecordSystem.Core/Models/ProfessionalStatusStatus.cs

```diff
@@ -12,7 +12,7 @@ public enum ProfessionalStatusStatus
         inductionExemption: FieldRequirement.NotApplicable,
         trainingProvider: FieldRequirement.Optional,
         degreeType: FieldRequirement.Optional,
-        country: FieldRequirement.Optional,
+        country: FieldRequirement.Mandatory,
```

Comment:
These seem to have changed on the spreadsheet, although I don't think it was specifically called out